provider/cmd/run.go

```diff
@@ -277,6 +280,11 @@ func RunCmd() *cobra.Command {
 		return nil
 	}
 
+	cmd.Flags().String(FlagProviderConfig, "", "provider configuration file path")
```

Comment:
This flag should probably be optional. Otherwise, existing providers will need to create a file that lists no attributes in order to continue to get the same behavior today.

Reply:
Done